bindings/python/src/lib.rs

```diff
@@ -421,6 +440,72 @@ fn format_pyerr(err: od::Error) -> PyErr {
     }
 }
 
+/// options in a write operation, shared by Operator.write() and AsyncOperator.write()
+struct WriteOptions {
```

Comment:
Can we reuse `OpWrite` instead? I perfer to have something like `parse_op_write(kwargs: Option<&PyDict>)` so that we don't have to sync changes between OpWrite and WriteOptions.

Reply:
Ok I think I managed to read `PyDict` into a `OpWrite`.

One small dissatisfaction from me is, I had to apply each option in my OpWrite to underlying `FunctionWrite` or `FutureWrite`, to finally mutate OpWrite in them.

If the underlying code accepts an OpWrite directly (I imagine a new method like `write_with_initial_op()`), the cost to apply each option and keep them consistent in future can be reduced.

Reply:
No problem 👍🏽